src/components/stimuli/Stimuli.tsx

```diff
@@ -12,9 +17,16 @@ const generateCards = (stimuliUrls: Array<string>): Array<JSX.Element> => {
   return cards;
 };
 
+const DEFAULT_FILE_INPUT_TEXT = 'Choose an image...';
```

Comment:
```suggestion
const DEFAULT_FILE_INPUT_TEXT = 'Select a file to upload';
```

We should make the prompt text consistent with the prompt on the Experiments page